tests/requirement/test_requirement_endo_bleeding.py

```
@@ -7,7 +7,8 @@
     RequirementType,
     FindingIntervention,
     ExaminationIndication,
-    PatientExamination
+    PatientExamination,
+    PatientExaminationIndication # Added import
```

Ruff (0.11.9)

`endoreg_db.models.PatientExaminationIndication` imported but unused

Remove unused import: `endoreg_db.models.PatientExaminationIndication`

(F401)